src/components/footer/Footer.tsx

```diff
@@ -15,7 +16,7 @@ const FooterContainer = styled.footer`
   }
 `;
 
-const Footer = () => {
+export const Footer = (): ReactElement => {
```

Comment:
Here you can do

`export const Footer:  React.FC = () => `

instead of assigning a return type of ReactElement. This is because we're saying footer is a FC (functional component) and FCs already return ReactElements. So if we type footer itself, we don't have to type its return type separately. The same for other components typed this way.